Eastern-Research-Group/expert-query
# Feature/65 add endpoints

## Related Issues:
* [EQ-65](https://jira.epa.gov/browse/EQ-65)
* [EQ-66](https://jira.epa.gov/browse/EQ-66)
* [EQ-67](https://jira.epa.gov/browse/EQ-67)

## Main Changes:
* Added endpoints for the assessmentUnits, assessmentUnitsMonitoringLocaitions, and sources profiles.
* Fixed an issue where the output column names didn't match up with the aliases, when the `columns` parameter is not provided. 

## Steps To Test:
1. Navigate to the following: 
    * http://localhost:9090/attains/data/assessmentUnits?id=1&id=2&id=3&id=4
    * http://localhost:9090/attains/data/assessmentUnitsMonitoringLocations?id=1&id=2&id=3&id=4
    * http://localhost:9090/attains/data/gisAssessments?id=1&id=2&id=3&id=4
    * http://localhost:9090/attains/data/sources?id=1&id=2&id=3&id=4
2. For the above output, verify the output is correct (all available columns included, camelCased column names, etc.)
3. Navigate to the following:
    * http://localhost:9090/attains/data/assessmentUnits?id=1&id=2&id=3&id=4&columns=assessmentUnitId&columns=assessmentUnitName&columns=locationDescription
    * http://localhost:9090/attains/data/assessmentUnitsMonitoringLocations?id=1&id=2&id=3&id=4&columns=assessmentUnitId&columns=assessmentUnitName&columns=locationDescription
    * http://localhost:9090/attains/data/gisAssessments?id=1&id=2&id=3&id=4&columns=assessmentUnitId&columns=assessmentUnitName&columns=organizationName
    * http://localhost:9090/attains/data/sources?id=1&id=2&id=3&id=4&columns=assessmentUnitId&columns=assessmentUnitName&columns=locationDescription
4. Repeat step 2
